Lavanya.C.Sharp/Set1Task6/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿
```

Comment:
Run the app and share the console screenshot in the comments.

Reply:
I posted output for both Set1Task6 and Set1Task7 in the comments.